docs/source/download.rst

```diff
@@ -141,177 +177,42 @@ Otherwise, use the appropriate values of ``PETSC_DIR`` and ``PETSC_ARCH``.
    It is not necessary that h5py be built with MPI support, although
    Firedrake supports both options.
 
+Further dependencies
+~~~~~~~~~~~~~~~~~~~~
 
-Potential installation errors on Mac OS
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Firedrake depends on the Python packages PyOP2_, FFC_, FIAT_ and UFL_.
```

Comment:
PyOP2 install page is now removed, but PyOP2 dependencies like COFFEE are not mentioned.